### ISSUE TITLE
GRCOpt.nw causes compilation to fail

### ISSUE THREAD

**Issue by cooljeanius** [cooljeanius/cec#1]
The error message with clang is:

```
GRCOpt.nw:361:17: error: no matching function for call to 'contains'
    if ( *j && !contains(levels, j - sync->successors.begin()) ) {
                ^~~~~~~~
```

and with gcc it is:

```
GRCOpt.nw: In constructor ‘GRCOpt::Simulator::Simulator(AST::GRCgraph&)’:
GRCOpt.nw:361: error: no matching function for call to ‘contains(std::set<int, std::less<int>, std::allocator<int> >&, ptrdiff_t)’
make[3]: *** [GRCOpt.o] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

This has been an issue since I first found this code... will probably need to contact upstream about it...

**Comment by fsantanna** [comment]
Hi! Any progress here?
I'm stuck in the same place. :)

**Comment by cooljeanius** [comment]
Nope, I kinda forgot about this, sorry...
(It's been a while since I've been able to use a browser that actually lets me check my notifications on here; I've been having hard drive issues...)

**Comment by gbormann** [comment]
See my pull-request... :-)

**Comment by cooljeanius** [comment]
> See my pull-request... :-)

You mean #2? I replied there...

**Comment by cooljeanius** [comment]
ok fixed by #2
